src/WebJobs.Extensions.Http/HttpConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    internal static class HttpConstants
```

Comment:
Add to the already existing constants class https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.Http/HttpExtensionConstants.cs